rosdep/python.yaml

```diff
@@ -964,6 +964,12 @@ python-gdown-pip:
   ubuntu:
     pip:
       packages: [gdown]
+python-genshi:
+  debian:
```

Comment:
these are all the alive debian distros so these rules can be collapsed to: `debian: [python-genshi]` (like you did for ubuntu)